.kitchen.yml

```diff
@@ -30,7 +30,7 @@ provisioner:
   salt_install: bootstrap
   salt_version: latest
   salt_bootstrap_url: https://bootstrap.saltstack.com
-  salt_bootstrap_options: -X -p rsync stable <%= version %>
+  salt_bootstrap_options: -X -p rsync git <%= version %>
```

Comment:
Is this line for the salt install method, or the version of the bootstrap script?

Reply:
Never mind. I think that is for the salt install method.